### ISSUE TITLE
Some release note between NUnit 2.6.4 and 3.14.0 should state the breaking change related to Trace and Debug output

### ISSUE THREAD

**Issue by maettu-this** [nunit/nunit#4685]
This issue is extracted from https://github.com/TestCentric/testcentric-gui/issues/1043, as it relates to the NUnit release notes, not TestCentric.

In the NUnit 2.x GUI, `Tools > Settings... > Gui > Text Output` provided an option to include Trace (and Debug) output. Learning from the discussion this is no longer supported by NUnit 3.x. https://github.com/nunit/nunit3-vs-adapter/issues/718 is about the very same topic. Apparently some people miss this functionality.

In my opinion this is a breaking change that should be documented in the corresponding release notes. When upgrading to NUnit 3.14.0 I did take the time to read through the complete list of framework release notes since NUnit 2.6.4. I may have missed it, but I think there is no change related to the handling of Trace and Debug output documented.

**Comment by OsirisTerje** [comment]
Feel free to raise a PR in the docs repo for this.

**Comment by maettu-this** [comment]
Would like to, but quite tricky to find the correct release. E.g. https://github.com/nunit/nunit3-vs-adapter/issues/301 states "the Breaking Change between NUnit 2.6.3 and NUnit 3.5.0". I assume it is either [NUnit 3.0.0 Alpha 1 (3.0.5378) - September 22, 2014] stating "Console Issues Resolved (Old nunit-console project, now combined with nunit)" or [NUnit 2.9.7 - August 8, 2014] stating "Breaking Changes" for the first time. No clue how to find the correct release though.

Let me at least propose the text here:

### Breaking Changes

- The NUnit GUI has been removed. Former nunit-console.exe now is nunit.exe.
- Trace and Debug output is no longer written to the NUnit output. A ConsoleTraceListener can be use to redirect Trace and Debug output to the NUnit output.

**Comment by CharliePoole** [comment]
AFAIK there is no longer an `nunit.exe` just `nunit3-console.exe`. Or did I forget? :-)

One memory I do have is that we had, at the time 3.0 was released or shortly thereafter, a combined set of release notes, which showed all the differences between 2.6.4 and 3.0. I wrote code so you could click and select whether to see the release notes as one group of changes or split among the various pre-releases.  I think that was left behind when we set up the new documentation but I can look to see if there's a copy in my archives.

@maettu-this Sorry this is so hard. You are trying to absorb about 18 years of change all at once. The rest of us got it in small pieces. :-)

**Comment by maettu-this** [comment]
> no longer an nunit.exe just nunit3-console.exe

You are right, only that it still was called `nunit-console.exe` in Alpha 1. ["Old nunit-console project, now combined with nunit"](https://docs.nunit.org/articles/nunit/release-notes/Pre-3.5-Release-Notes.html#console-issues-resolved-old-nunit-console-project-now-combined-with-nunit) let me assume that `nunit-console` got removed then. I'll fix the release note proposed above.

> about 18 years of change all at once

Actually it is just 10 years, 2.6.4 was in 2014. Before I used 2.5.10, 2.5.7, 2.5.2, 2.4.6. And in 2017 I attempted to upgrade to NUnit 3.x but failed for a bunch of reasons. Since then the priority was on other parts of the software. Having some room momentarily let me try again :-)

**Comment by OsirisTerje** [comment]
As we don't have a specific release note section for this, I'll move this into the How-To category in Discussions, so other people can pick it up if they do the same transition.
